### ISSUE TITLE
How to update login view when user name or password wrong

### ISSUE THREAD

**Issue by letlet** [chaione/Cely#15]
I want to tell the user name or password is wrong when there is something wrong with the user's input by update login view

**Comment by initFabian** [comment]
Hey @letlet,

Thanks for the issue, this is a great suggestion! 

The thing about this topic is that this trends the borders of just creating a custom LoginViewController. Before just classifying as a custom LoginVC need, can you give me some more details... Would a simple `AlertViewController` popup work?

**Comment by letlet** [comment]
Yes, a AlertViewController popup works and is the solution I'm using. But it's not elegant. Error message above the input area is more usefull.

**Comment by initFabian** [comment]
Im still very open to this feature request. 

If someone doesn't mind, open up a PR that passes the following criteria:

- A [single error message](https://stackoverflow.com/questions/14922130/which-error-message-is-better-when-users-entered-a-wrong-password) appearing onto of the Username textfield
- Add styling option in [`CelyStyle`](https://github.com/chaione/Cely/blob/master/Sources/CelyProtocols.swift#L25) to change `Font` of styling, but have a default implementation inside of CelyStyle `extension`.
- Update Documentation and affected tests

**Comment by rlam3** [comment]
@letlet did you find a workaround for this? I'm running into a similar situation with cely...
